src/wallet/scriptpubkeyman.cpp

```diff
@@ -1486,11 +1486,6 @@ bool DescriptorScriptPubKeyMan::CanGetAddresses(bool internal)
     return false;
 }
 
-bool DescriptorScriptPubKeyMan::Upgrade(int prev_version, std::string& error)
```

Comment:
In  e40a833c3bffb0cf723357238eb809398e3e5237 `Create LegacyScriptPubKeyMan when not a descriptor wallet`: accidentally dropping `Upgrade()`?

Reply:
It should be dropped, but not in that commit. Moved it to the dummy class definition.